.github/actions/docker/action.yaml

```diff
@@ -42,81 +52,38 @@ runs:
         password: ${{ inputs.dockerhub_token }}
       if: ${{ inputs.dockerhub_token != '' }}
 
-    - name: Set up QEMU
-      uses: docker/setup-qemu-action@v3
-      with:
-        platforms: ${{ inputs.platforms }}
-
-    - name: Format platforms for docker
-      uses: actions/github-script@v6
-      id: platforms
-      with:
-        result-encoding: string
-        script: |
-          return '${{inputs.platforms}}'
-             .split(',')
-             .map((platform) => `linux/${platform.trim()}`)
-             .join(',')
-
     - name: Set up Docker BuildX
       id: buildx
       uses: docker/setup-buildx-action@v3
       with:
         install: true
         driver-opts: env.BUILDKIT_STEP_LOG_MAX_SIZE=10485760
         cleanup: false
-        platforms: ${{ steps.platforms.outputs.result }}
         config-inline: |
           [worker.oci]
-            gc = true
-            gckeepstorage = 40000 # 40 GB
-            [[worker.oci.gcpolicy]]
-              all = true
-              keepBytes = 30000000000 # 30 GB
-              keepDuration = 864000 # 10 days
+            gc = false
```

Comment:
it means the cache size will grow indefinitely

Reply:
We drop it every time. Only cargo registry mount is stored on S3 where we have own retention policy for files (3 days)